src/pages/Home.css

```diff
@@ -0,0 +1,81 @@
+/* MOVIE POSTER GRID */
+
+.home-container {
+    background-color: black;
+    margin: -10px;
+}
+
+.movie-boxes {
+    display: grid;
+    grid-template-columns: 1fr;
+}
+
+.movie-poster {
+    width: 100%;
+    margin: -2px;
+}
+
+
+/* MOVIE DETAILS VISIBLE ON HOVER */
+
+.movie-box:hover {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    position: relative;
+}
+
+.movie-box .hover-info {
+    position: absolute;
+    padding: 25px;
+    display: none;
+    margin-bottom: -10px;
+}
+
+.movie-box:hover .hover-info {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    bottom: 0;
+    left: 0;
+    line-height: 40px;
+    color: #fff;
+    text-decoration: none;
+    transition: 0.3s ease;
+}
+
+.movie-poster:hover {
+    opacity: 15%;
+    transition: 0.3s ease;
+}
+
+.title {
+    font-weight: bold;
+    margin-bottom: -12px;
+}
+
+.release-date {
+    font-weight: normal;
+    color: #b3b3b3;
+}
+
+
+/* MEDIA QUERIES */
+
+@media (min-width: 500px) AND (max-width: 699px) {
+    .movie-boxes {
+        grid-template-columns: 1fr 1fr;
+    }
+}
+
+@media (min-width: 700px) AND (max-width: 999px) {
+    .movie-boxes {
+        grid-template-columns: 1fr 1fr 1fr;
+    }
+}
+
+@media (min-width: 1000px) {
+    .movie-boxes {
+        grid-template-columns: 1fr 1fr 1fr 1fr;
```

Comment:
Good that there's control of how many movies are shown and that the margin/gap (which is 0) is kept consistent.